docker/docker_image_dest.go

```diff
@@ -17,6 +17,21 @@ import (
 	"github.com/pkg/errors"
 )
 
+var manifestMIMETypes = []string{
+	// TODO(runcom): we'll add OCI as part of another PR here
+	manifest.DockerV2Schema2MediaType,
+	manifest.DockerV2Schema1SignedMediaType,
+	manifest.DockerV2Schema1MediaType,
+}
+
+func supportedManifestMIMETypesMap() map[string]bool {
```

Comment:
I think a `map[string]struct{}` is more idiomatic, but I really don’t care enough.